src/osyris/plot/wrappers.py

```diff
@@ -126,3 +126,50 @@ def scatter(ax, x, y, data, **kwargs):
         return coll
     else:
         return ax.scatter(x, y, **default_args)
+
+def line_integral_convolution(ax, x, y, z, **kwargs):
+    """
+    Wrapper that plots a line integral convolution of a vector field.
+    Uses alpha blending to merge the LIC with a user-requested color.
+    """
+    import lic as lic
+    from matplotlib.cm import ScalarMappable
+    from matplotlib.colors import LogNorm
+
+    #compute line integral convolution
+    try:
+        lic_res = lic.lic(z[...,1], z[...,0], length=kwargs["length"])
+        del kwargs["length"]
+    except KeyError:
+        lic_res = lic.lic(z[...,1], z[...,0], length=30)
+
+    plot_args = {**kwargs}
+    plot_args["extent"] = [np.min(x),np.max(x),np.min(y),np.max(y)]
+    plot_args["origin"] = "lower"
+
+    alpha_blending = True
+    try:
+        del plot_args["color"]
+    except KeyError:
+        alpha_blending = False
+
+    if alpha_blending:
+        plot_args_color = {**plot_args}
+        plot_args_color["norm"] = LogNorm()
+        plot_args_color["alpha"] = 1.
+        ax.imshow(z[...,2], **plot_args_color)
+
+        plot_args["alpha"] = .3
+
+    #amplify contrast on lic
+    lim=(.2,.5)
+    lic_data_clip = np.clip(lic_res,lim[0],lim[1])
+    lic_data_rgba = ScalarMappable(norm=None, cmap="binary").to_rgba(lic_data_clip)
+    lic_data_clip_rescale = (lic_data_clip-lim[0])/(lim[1]-lim[0])
+    lic_data_rgba[...,3] = lic_data_clip_rescale * 1
```

Comment:
Why is the `* 1` needed here?

Reply:
It's not, I was fiddling with some values and not changing lic_data_clip_rescale was fine as is.